pkg/credentialprovider/gcp/metadata.go

```diff
@@ -121,7 +121,8 @@ func onGCEVM() bool {
 		glog.V(2).Infof("Error while reading product_name: %v", err)
 		return false
 	}
-	return strings.Contains(string(data), googleProductName)
```

Comment:
Delete the `googleProductName` variable declared at the beginning of the file.

Reply:
Ack will do and update the PR later today